{"metadata":{"role":"symbol","symbolKind":"case","title":"BluetoothServiceError.notConnected(_:)","roleHeading":"Case","externalID":"s:9VigramSDK21BluetoothServiceErrorO12notConnectedyAC10Foundation4UUIDVcACmF","modules":[{"name":"VigramSDK"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"notConnected","kind":"identifier"},{"kind":"text","text":"("},{"preciseIdentifier":"s:10Foundation4UUIDV","text":"UUID","kind":"typeIdentifier"},{"kind":"text","text":")"}]},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"kind":"symbol","identifier":{"url":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK","doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError"]]},"abstract":[{"type":"text","text":"Connecting to a device failed unexplicably."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"notConnected","kind":"identifier"},{"kind":"text","text":"("},{"text":"UUID","preciseIdentifier":"s:10Foundation4UUIDV","kind":"typeIdentifier"},{"text":")","kind":"text"}],"platforms":["iOS"],"languages":["swift"]}]}],"references":{"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError":{"title":"BluetoothServiceError","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothserviceerror","kind":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"BluetoothServiceError"}],"role":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError","navigatorTitle":[{"kind":"identifier","text":"BluetoothServiceError"}],"abstract":[{"type":"text","text":"BluetoothServiceError lists different errors that can be propagated by "},{"type":"reference","isActive":true,"identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService"},{"type":"text","text":" implementations."}]},"logo.png":{"variants":[{"traits":["1x","light"],"url":"\/images\/com.vigram.VigramSDK\/logo.png"}],"alt":"Vigram logo.","type":"image","identifier":"logo.png"},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothServiceError/notConnected(_:)":{"type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothServiceError\/notConnected(_:)","abstract":[{"text":"Connecting to a device failed unexplicably.","type":"text"}],"kind":"symbol","url":"\/documentation\/vigramsdk\/bluetoothserviceerror\/notconnected(_:)","title":"BluetoothServiceError.notConnected(_:)","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"notConnected"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:10Foundation4UUIDV","text":"UUID"},{"text":")","kind":"text"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK/BluetoothService":{"fragments":[{"kind":"keyword","text":"protocol"},{"kind":"text","text":" "},{"text":"BluetoothService","kind":"identifier"}],"title":"BluetoothService","kind":"symbol","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK\/BluetoothService","type":"topic","url":"\/documentation\/vigramsdk\/bluetoothservice","role":"symbol","navigatorTitle":[{"kind":"identifier","text":"BluetoothService"}],"abstract":[{"text":"A BluetoothService is used to handle bluetooth connections.","type":"text"}]},"doc://com.vigram.VigramSDK/documentation/VigramSDK":{"role":"collection","title":"VigramSDK","kind":"symbol","images":[{"identifier":"logo.png","type":"icon"}],"abstract":[],"url":"\/documentation\/vigramsdk","type":"topic","identifier":"doc:\/\/com.vigram.VigramSDK\/documentation\/VigramSDK"}}}